tools/UpdatePaths/__main__.py

```diff
@@ -13,6 +13,7 @@
     /turf/open/floor/iron/warningline : /obj/effect/turf_decal {@OLD} , /obj/thing {icon_state = @OLD:name; name = "meme"}
     /turf/open/floor/iron/warningline{dir=2} : /obj/thing
     /obj/effect/landmark/start/virologist : @DELETE
+    /mob/living{resize = @ANY} : /mob/living{@OLD; resize = @SKIP, current_size = @OLD:resize}
```

Comment:
i think you should write a small blurb on what the colon within the curly braces here does because although the readme is pretty neat, someone who just wants the cliffnotes on what a thing does should only have to read a sentence